### PR TITLE
Prepared new release ``v0.12.0``

### PR DESCRIPTION
**Pull request by behackl**
Preparations for new release.

Link to changelog: https://manimce--2247.org.readthedocs.build/en/2247/changelog/0.12.0-changelog.html

### CODE REVIEW

**Comment by behackl** [comment]
At this point I'm happy with the changelog; I don't think a further regeneration will be necessary. Minor errors (typos and stuff) could still be fixed (by changing the rst file manually).

Needs reviews!

**Review by Darylgolden — APPROVED** [review]
LGTM!

**Review by christopher-besch — APPROVED** [review]

**Review by kolibril13 — APPROVED** [review]

File: docs/source/changelog/0.12.0-changelog.rst [hunk 1]
```diff
@@ -0,0 +1,258 @@
...
+
+* `#2196 <https://github.com/ManimCommunity/manim/pull/2196>`__: Implemented :meth:`.Line3D.parallel_to` and :meth:`.Line3D.perpendicular_to`
+
+
+Enhancements
+------------
+
+* `#2138 <https://github.com/ManimCommunity/manim/pull/2138>`__: Fixed example for :meth:`~.Vector.coordinate_label` and added more customization for :class:`~.Matrix`
+   - Added `**kwargs` for :meth:`~.Vector.coordinate_label`.
+   - :class:`~.Matrix` now accepts a `bracket_config` parameter.
```
Comment (behackl) [inline docs/source/changelog/0.12.0-changelog.rst:121]:
```suggestion
   - :class:`~.Matrix` now accepts a ``bracket_config`` keyword argument.
```

File: docs/source/changelog/0.12.0-changelog.rst [hunk 2]
```diff
@@ -0,0 +1,258 @@
...
+
+
+* `#2196 <https://github.com/ManimCommunity/manim/pull/2196>`__: Implemented :meth:`.Line3D.parallel_to` and :meth:`.Line3D.perpendicular_to`
+
+
+Enhancements
+------------
+
+* `#2138 <https://github.com/ManimCommunity/manim/pull/2138>`__: Fixed example for :meth:`~.Vector.coordinate_label` and added more customization for :class:`~.Matrix`
+   - Added `**kwargs` for :meth:`~.Vector.coordinate_label`.
```
Comment (behackl) [inline docs/source/changelog/0.12.0-changelog.rst:120]:
```suggestion
   - Additional keyword arguments for :meth:`~.Vector.coordinate_label` are passed to the constructed matrix.
```

**Review by behackl — COMMENTED** [review]
Two minor improvements for detailed explanations, should be good to go then.

**Comment by behackl** [comment]
Thanks everyone!

**Pull request merged**